deepinv/physics/forward.py

```diff
@@ -223,6 +223,22 @@ def A_dagger(self, y, x_init=None):
 
         return x.clone()
 
+    def A_jvp(self, x, v):
+        r""" "
+        Applies the Jacobian of the forward operator to the input x and perturbation v, defined as:
```

Comment:
Computes the product between the Jacobian of the forward operator :math:`A` evaluated at :math:`x` and the vector :math:`v`, defined as: